openapi/data/fields.py

```diff
@@ -234,16 +234,11 @@ def __call__(self, field, value, data=None):
 
 class NumberValidator(Validator):
 
-    def __init__(self, min_value=None, max_value=None, precision=None):
+    def __init__(self, min_value=None, max_value=None):
         self.min_value = min_value
         self.max_value = max_value
-        self.precision = precision
 
     def __call__(self, field, value, data=None):
-        try:
-            value = round(value, self.precision)
```

Comment:
we still need to test if it is a valid number right?
```python
value = float(value)
```

Reply:
I've moved this out of `NumberValidator` and into `DecimalValidator` now so we're doing a `Decimal` conversion before round